docker/STAR/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM ubuntu:15.04
+MAINTAINER Andrey V Kartashov "porter@porter.st"
+COPY STAR /usr/local/bin/STAR
```

Comment:
Indeed! The pre-packaged software is a preference, not a requirement. Gotta use what is available to get the job done :-)